helpers.js

```diff
@@ -36,3 +36,16 @@ export const formatDateDiff = (recent, old) => {
     day: "numeric",
   }).format(old.getTime()); // ex: Nov 15
 };
+
+export function formatLargeNumber(n) {
+  let s = "";
+  if (n < 1000) {
+    s = n;
```

Comment:
The function must return `string`.
When `n` is < than 1000, the function returns an `integer`.